### ISSUE TITLE
[BUG] Footer links lack sufficient contrast

### ISSUE THREAD

**Issue by sublimemarch** [ContributorCovenant/beacon#65]
All contributions, including pull requests, issues, and comments, are governed by our [code of conduct](https://github.com/ContributorCovenant/beacon/blob/release/CODE_OF_CONDUCT.md).

## Description of the bug
Links within the footer don't have enough contrast, per [WCAG 2.1 AA guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html). This was determined using [Axe](https://www.deque.com/axe/) and can be verified using any color contrast checking tool, such as the [WebAIM color contrast checker](https://webaim.org/resources/contrastchecker/).

Footer links have a contrast of 3.23 currently. The expected contrast ratio for text of this size is 4.5.

## Expected behavior
The colors of these elements should be updated to have more contrast. Alternatively, a bolder font or larger font size could be used, but I'd recommend updating the colors themselves. This is a necessary accessibility improvement for people with disabilities (such as color blindness and low-vision).

## Screenshots
This applies to the orange link text against dark blue:
![image](https://user-images.githubusercontent.com/9502152/52145915-edfef580-2627-11e9-971a-884b7d9ca70f.png)
